### ISSUE TITLE
[v1.2.10] Setting showThumbnail=false breaks the ImageGallery

### ISSUE THREAD

**Issue by ethanngoh** [xiaolin/react-image-gallery#703]
**Describe the bug**\
When one sets `showThumbnail={false}`, the component errors out: 

```
react-dom.development.js:22839 
        
       Uncaught TypeError: Failed to execute 'observe' on 'ResizeObserver': parameter 1 is not of type 'Element'.
    at l.value (image-gallery.js:1:1)
    at l.value (image-gallery.js:1:1)
    at commitLayoutEffectOnFiber (react-dom.development.js:23305:1)
    at commitLayoutMountEffects_complete (react-dom.development.js:24688:1)
    at commitLayoutEffects_begin (react-dom.development.js:24674:1)
    at commitLayoutEffects (react-dom.development.js:24612:1)
    at commitRootImpl (react-dom.development.js:26823:1)
    at commitRoot (react-dom.development.js:26682:1)
    at finishConcurrentRender (react-dom.development.js:25981:1)
    at performConcurrentWorkOnRoot (react-dom.development.js:25809:1)
```
**Image Gallery Version**\
What version of `react-image-gallery` are you using?
1.2.10

**To Reproduce**\
1. npx create-react-app test-app --template typescript
2. Paste example code into App.tsx:
```
import React from "react";
import ImageGallery from "react-image-gallery";
import "./App.css";


const images = [
  {
    original: "https://picsum.photos/id/1018/1000/600/",
    thumbnail: "https://picsum.photos/id/1018/250/150/",
  },
  {
    original: "https://picsum.photos/id/1015/1000/600/",
    thumbnail: "https://picsum.photos/id/1015/250/150/",
  },
  {
    original: "https://picsum.photos/id/1019/1000/600/",
    thumbnail: "https://picsum.photos/id/1019/250/150/",
  },
];

function App() {
  return <ImageGallery items={images} showThumbnails={false} />;
}

export default App;
```

**Expected behavior**\
The console should not show errors and the component should display.

**Screenshots**\

**Client info (please complete the following information):**\
 - OS: windows
 - Browser: [chrome, firefox, edge]

**Additional context**\
version 1.2.8 did not have this issue.

Potentially related to this change: https://github.com/xiaolin/react-image-gallery/commit/838d742b40fe1cd22045fd4a1096ecfa785f886a#diff-8423737b164d0bb028cee16f4aad794d4d19a5b2313dcc2f1130501b7f04267fR1012

**Comment by xiaolin** [comment]
Thanks for reporting, I've published a fix with version 1.2.11.

**Comment by ethanngoh** [comment]
Thanks for the quick turnaround, looks like it's been fixed!
